### ISSUE TITLE
Improve help tooltip for "Revenue per visit" column metric

### ISSUE THREAD

**Issue by erodrig9** [matomo-org/matomo#8495]
I am seeing incorrect values for the "Revenue Per Visit" column in the Referrer Websites widget. However the values in "Goal Ecommerce conversions by type of visit" look correct. I attached screen shots of the two for comparison. All other fields look good except for this one.
![ecommerceconversions2](https://cloud.githubusercontent.com/assets/788857/9093867/a0552528-3b7c-11e5-8fa3-7243b2a52b0f.PNG)
![referrerwebsites2](https://cloud.githubusercontent.com/assets/788857/9093868/a0562f04-3b7c-11e5-9c23-9a463ea80730.png)

**Comment by mattab** [comment]
Hi @erodrig9 

> I am seeing incorrect values for the "Revenue Per Visit" column in the Referrer Websites widget. 

"Revenue per visit" in the referrer websites widget, also includes the sum of revenue for all Goals.

Whereas in Ecommerce "Revenue per visit" it only includes the Ecommerce revenue.

To do:
 * maybe we could improve "Revenue per visit" and "Conversion rate" columns metric tooltip help text, to make it clear which Goal(s) this refers to

**Comment by erodrig9** [comment]
@mattab My website does not have any other types of goals setup and the referrer widget still shows a positive value for revenue per visit.  We are still on version 2.7.0, but we are planning to update to the latest version soon.

I attached screen shots.


![goals](https://cloud.githubusercontent.com/assets/788857/10672065/c4411ae2-78bb-11e5-8fad-c9b20dd894c7.PNG)
![referrer](https://cloud.githubusercontent.com/assets/788857/10672066/c4898908-78bb-11e5-810a-4947f21f2c52.PNG)

**Comment by mattab** [comment]
@erodrig9 try to upgrade to 2.15.0 where this issue may be fixed

**Comment by erodrig9** [comment]
@mattab We would like to confirm that the issue is fixed before upgrading our production environment.  Is there a way to make sure the latest version will have this resolved?

**Comment by erodrig9** [comment]
@mattab Would it be possible to see if the latest version of Piwik fixes this issue with existing data or would we have to verify with new data after the upgrade?

**Comment by erodrig9** [comment]
@mattab I Updated a dev piwik environment to 3.0.3 where I saw this issue as well.  I deleted the archive data and let 3.0.3 archive the yearly data, however I am still seeing this issue.  

The Goals by Website widget is showing the revenue correctly.  However, the Referrers website still shows a value for revenue even though there are no ecommerce or goal conversions.  We don't use any other goals other than ecommerce.

![all referrers](https://cloud.githubusercontent.com/assets/788857/25757135/63414762-3197-11e7-8cb6-7ab5cc4eaedd.PNG)

![goalsbywebsites](https://cloud.githubusercontent.com/assets/788857/25757140/6784116a-3197-11e7-88e2-00cbc2534985.PNG)
